SwiftUI_DZ/SwiftUI_DZ/ContentView.swift

```diff
@@ -8,15 +8,96 @@
 import SwiftUI
 
 struct ContentView: View {
+    // MARK: - Private Constants
+    private enum Constants {
+        static let exampleSimpleAlertString = "Example Alert"
+        static let exampleComplexAlertString = "Example Alert  with two buttons and logic"
+        static let exampleSimpleActionSheetString = "Example ActionSheet"
+        static let exampleComplexActionSheetString = "Example ActionSheet with two buttons and logic"
+        static let showString = "Show"
+        static let titleString = "Alarm"
+        static let textString = "Text changed"
+        static let sureString = "you are sure?"
+        static let okString = "OK"
+        static let colorString = "change color?"
+        static let spacingSGFloat: CGFloat = 5.0
+        
+    }
+    
+    // MARK: - Private Properties
+    
+    @State private var isTapSimpleAlert = false
+    @State private var isTapComplexAlert = false
+    @State private var isTapSimpleAction = false
+    @State private var isTapComplexAction = false
```

Comment:
```suggestion
    @State private var isTapComplexAction = false
```
Все приватки ниже var body